### DISCUSSION TITLE
为什么大部分文件搜索不了？

### DISCUSSION THREAD

**Discussion by zjrwtx** [spencerwooo/onedrive-vercel-index#702] [Q&A]
![image](https://user-images.githubusercontent.com/86822589/174480658-b13301d9-3b60-4f2f-8adc-96c61cf6a861.png)
为什么大部分文件搜索不了？

**Comment by zjrwtx** [comment]
网站地址：https://zjrwtxsstorage.vercel.app/zh-CN/

**Reply by aylitat** [reply]
解决了吗？我发现我的也是这样
